packages/@o3r/localization/src/tools/localization.service.ts

```diff
@@ -106,7 +106,7 @@ export class LocalizationService {
    * @param interpolateParams Object to use in translation binding
    * @returns A stream of the translated key
    */
-  private getTranslationStream(translationKey: string, interpolateParams?: object) {
```

Comment:
why do you need to expose this field instead of using `translate` ?

Reply:
I did not caught translate emitted on language changes, I will revert it